hurumap_land/templates/profile/profile_detail.html

```diff
@@ -346,6 +346,42 @@ <h1>Individual Land</h1>
                 data-source-link="http://www.ruraldevelopment.gov.za/publications/land-audit-report/file/6126"></div>
             </div>
         </section>
+        <section class="clearfix stat-row">
```

Comment:
Add those `if` statements to check presence of data before plotting the charts, otherwise we'll end up with those unwanted *empty* sections.